core/assets/js/app.js

```diff
@@ -57,6 +57,18 @@ let csrfToken = document
   .querySelector("meta[name='csrf-token']")
   .getAttribute("content");
 let Hooks = {};
+
+Hooks.NativeWrapper = {
+  mounted() {
+    window.nativeWrapperHook = this
```

Comment:
I don't see the usage of `nativeWrapperHook`. Am I overlooking something?

Reply:
This is in EyraUI.Navigation.Button

```
      <div id={{@id}} phx-hook="NativeWrapper" @click="nativeWrapperHook.toggleSidePanel(); {{ @overlay }}" >
        <Get :if={{ @method === :get && !is_dead?(@action)}} path={{@action.target}}>
```